### PR TITLE
Remove selective filter tutorial

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/how-tos/deploy-ml.md [hunk 1]
```diff
@@ -27,6 +28,8 @@ You can use Viam's built-in tools to train a machine learning (ML) model on your
 
 ![Diagram of the camera component to data management service to ML model service to vision service pipeline.](/how-tos/ml-vision-diagram.png)
 
+You can use ML models to help your machines adapt their behavior to the world around them.
```
Comment (JessamyT) [inline docs/how-tos/deploy-ml.md:31]: This is part of #3410 also?

Comment (npentrel) [reply docs/how-tos/deploy-ml.md]: hadn't rebased yet

File: docs/how-tos/deploy-ml.md [hunk 2]
```diff
@@ -89,18 +90,24 @@ Start by assembling the dataset to train your machine learning model on.
...
-1. Unzip the download.
-1. Open a terminal and go to the dataset folder.
-1. In it you will find a Python script to upload the data to the Viam app.
-1. Open the script and fill in the constants at the top of the file.
-1. Run the script to upload the data into a dataset in Viam app:
+We have two datasets you can use for testing, one with shapes and the other wise a wooden figure:
+
+{{<imgproc src="/tutorials/data-management/shapes-dataset.png" resize="1200x" declaredimensions=true style="max-width:400px" alt="The shapes dataset." class="imgzoom fill aligncenter">}}
+
+{{< imgproc src="/tutorials/filtered-camera-module/viam-figure-dataset.png" style="max-width:400px" alt="The datasets subtab of the data tab in the Viam app, showing a custom 'viam-figure' dataset of 25 images, most containing the wooden Viam figure" class="imgzoom fill aligncenter" resize="1400x" >}}
```
Comment (npentrel) [inline docs/how-tos/deploy-ml.md:97]: Note to self this should also get added to the datasets page

File: docs/how-tos/deploy-ml.md [hunk 3]
```diff
@@ -89,18 +90,24 @@ Start by assembling the dataset to train your machine learning model on.
...
+
+1. [Download the shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip) or [download the wooden figure dataset](https://storage.googleapis.com/docs-blog/dataset-figure.zip).
+2. Unzip the download.
+3. Open a terminal and go to the dataset folder.
+4. In it you will find a Python script to upload the data to the Viam app.
+5. Open the script and fill in the constants at the top of the file.
+6. Run the script to upload the data into a dataset in Viam app:
```
Comment (npentrel) [inline docs/how-tos/deploy-ml.md:104]:
```suggestion
1. [Download the shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip) or [download the wooden figure dataset](https://storage.googleapis.com/docs-blog/dataset-figure.zip).
1. Unzip the download.
1. Open a terminal and go to the dataset folder.
1. In it you will find a Python script to upload the data to the Viam app.
1. Open the script and fill in the constants at the top of the file.
1. Run the script to upload the data into a dataset in Viam app:
```

File: docs/how-tos/deploy-ml.md [hunk 4]
```diff
@@ -89,18 +90,24 @@ Start by assembling the dataset to train your machine learning model on.
...
 
    ```sh {class="command-line" data-prompt="$" }
    python3 upload_data.py
    ```
 
-1. Continue to [Train a machine learning model](#train-a-machine-learning-ml-model).
+7. Continue to [Train a machine learning model](#train-a-machine-learning-ml-model).
```
Comment (npentrel) [inline docs/how-tos/deploy-ml.md:110]:
```suggestion
1. Continue to [Train a machine learning model](#train-a-machine-learning-ml-model).
```

File: docs/how-tos/deploy-ml.md [hunk 5]
```diff
@@ -89,7 +90,13 @@ Start by assembling the dataset to train your machine learning model on.
 
 {{% expand "Just testing and want a dataset to get started with? Click here." %}}
 
-1. [Download our shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip).
+We have two datasets you can use for testing, one with shapes and the other wise a wooden figure:
```
Comment (JessamyT) [inline docs/how-tos/deploy-ml.md:93]:
```suggestion
We have two datasets you can use for testing, one with shapes and the other with a wooden figure:
```

File: docs/how-tos/deploy-ml.md [hunk 6]
```diff
@@ -129,9 +138,14 @@ The more varied the provided data set, the more accurate the resulting model bec
...
-Once you have enough images of the objects you'd like to classify, use the interface on the **DATA** tab to label your data.
-If you want to train an image classifier, use image tags.
-For an object detector, use bounding boxes.
+Once you have enough images of the objects you'd like to identify captured and synced to the Viam app, use the interface on the [**DATA** tab](https://app.viam.com/data/view) to label your data.
+
+You can label your images to create:
+
+- **Detection models**: Draw **bounding boxes** around distinct objects within captured images.
+  The trained model will enable your machine to be able to detect those objects on its own.
+- **Classification models**: Add **tags** to your images with class labels that describes it.
```
Comment (JessamyT) [inline docs/how-tos/deploy-ml.md:147]:
```suggestion
- **Classification models**: Add **tags** to each of your images with class labels that describe it.
```

File: docs/how-tos/deploy-ml.md [hunk 7]
```diff
@@ -360,6 +381,11 @@ Once your model has finished training, you can test it with images in the Viam a
 
 If the results exceed the confidence threshold, the **Run model** section shows a label and the responding confidence threshold.
 
+When satisfied that your ML model is working well, continue to [deploy an ML model](#deploy-an-ml-model).
+If the vision service is not matching reliably, you will need to adjust your ML model by consider adding and labelling more images in your dataset.
```
Comment (JessamyT) [inline docs/how-tos/deploy-ml.md:385]:
```suggestion
If the vision service is not detecting or classifying reliably, you will need to adjust your ML model by consider adding and labelling more images in your dataset.
```

File: docs/how-tos/deploy-ml.md [hunk 8]
```diff
@@ -360,6 +381,11 @@ Once your model has finished training, you can test it with images in the Viam a
...
+
+Ideally, you want your ML model to be able to identify objects with a high level of confidence, which is dependent on a robust source dataset.
```
Comment (JessamyT) [inline docs/how-tos/deploy-ml.md:387]:
```suggestion
Ideally, you want your ML model to be able to identify objects with a high level of confidence, which is dependent on a robust training dataset.
```
Opt

**Review by JessamyT — APPROVED** [review]
LGTM % typos

**Pull request merged**